crates/autopilot/src/run.rs

```diff
@@ -240,15 +229,12 @@ pub async fn run(args: Arguments) {
             UniV2BaselineSourceParameters::from_baseline_source(*source, &chain_id.to_string())
         })
         .chain(args.shared.custom_univ2_baseline_sources.iter().copied());
-    let (pair_providers, pool_fetchers): (Vec<_>, Vec<_>) = futures::stream::iter(univ2_sources)
+    let pair_providers: Vec<_> = futures::stream::iter(univ2_sources)
```

Comment:
This is still required for bad token detection? Do we have a plan somewhere on how to remove that beast as well?

Reply:
Yeah, the bad token detection still uses it. :/
I don't have a very concrete plan how to move forward with this. My idea was to use our currently strictest quote verification mode `enforce-if-possible` to phase out bad token detection. There are a bunch of caveats around this so I would just try it out on staging and see how it goes. 🙏